src/interfaces/widgets/button.py

```diff
@@ -0,0 +1,57 @@
+"""
+
+    This module generate a QPushButton object.
+
+"""
+
+from PyQt5.QtWidgets import QPushButton, QRadioButton, QWidget
+from PyQt5.QtGui import QIcon
+from PyQt5.QtCore import QSize, Qt
+
+
+class Button(QPushButton):
+
+    def __init__(self, widget:QWidget, **kwargs):
+        super(Button, self).__init__(widget)
+
+        # Lambda func grab input args
+        get_param = lambda x : kwargs.get(x)
+
+        text = get_param("text")
```

Comment:
What happens if any of these `kwargs` isn't provided?

Reply:
The button would contains no label. (If in case if we later on need some button only contain icons.